torchbenchmark/models/llama_v2/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 1
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+not_implemented:
+- jit: true
+- device: cpu
+- device: cuda
+  test: train
+- device: cuda
+  test: example
```

Comment:
I am wondering why cuda example is disabled? Is it because of OOM?

The example test will check whether the output tensor is deterministic across runs, and it will be useful for the accuracy tests.

Reply:
Yep at num_heads = 16 this was the only test that OOM'd because of some cloning - lemme get the error message

Reply:
Here you go https://github.com/pytorch/benchmark/actions/runs/5623600850/job/15238632991#step:6:909

Reply:
If cloning is an issue, you can add `DEEPCOPY = False` to disable it, similar to https://github.com/pytorch/benchmark/blob/36e1ed1c435ff8a4953a7c5e4791e76e6714b269/torchbenchmark/models/fambench_xlmr/__init__.py#L57

Can you please add `DEEPCOPY = False` and try again?

Reply:
Alright trying it now in CI

Reply:
Ok with deep copy off it still ooms at the full model size